integration/src/main/kotlin/kotlinx/benchmark/integration/ProjectBuilder.kt

```diff
@@ -23,7 +29,10 @@ benchmark {
         ${targets.flatMap { it.value.lines(it.key) }.joinToString("\n        ")}
     }
 }
-            """.trimIndent()
+kotlin {
+    ${kotlinConfig.lines().joinToString("\n    ")}
+}
+        """.trimIndent()
```

Comment:
Minor: It's a nice practice to write opening and closing triple quotes with the same indentation.

Reply:
Thank you for catching this.